ui/apps/platform/src/Containers/Clusters/ClusterLabelsTable.tsx

```diff
@@ -77,7 +77,7 @@ function ClusterLabelsTable({
                 <Tr>
                     <Th>Key</Th>
                     <Th>Value</Th>
-                    {hasAction && <Th aria-label="Action" />}
+                    {hasAction && <Td />}
```

Comment:
This seems unsemantic. Wouldn't having some kind of label be more appropriate a fix?

Reply:
Yes, good point. I will change it render visible **Action** table heading.